bucket/mousejiggler.json

```diff
@@ -1,19 +1,34 @@
 {
-    "version": "1.8.42",
+    "version": "2.0.9",
```

Comment:
2.0.13

Reply:
Fixed the version numbers.

Wrt the error shown, as I understand it it's possible to install the .NET 5 runtime on x64 without the x86 runtime included. If it wasn't included, that message is expected.

I retested here with a fresh x64 VM, a default .NET 5 Desktop runtime install, and the x86 download, and don't get it.

Reply:
So there is dotnet dependency and nowhere in manifest it is mentioned??

Reply:
There isn't a package for it to depend on, and since the .NET 5 runtime has to be installed globally, it wouldn't make a good Scoop package anyway.

(In theory I could build MJ as a self-contained app that would eliminate the dependency, but that turns a 1.4 MB executable into a 126 MB executable, and no-one wants that.)

Reply:
https://raw.githubusercontent.com/lukesampson/scoop-extras/master/bucket/windowsdesktop-runtime.json

Reply:
Huh. That did not show up when I searched for it. Curious.

On the other hand, that's also for 3.1, not 5.

I can make one for 5, readily enough, but I've got another consideration to bring up. The reason I'm setting up a Scoop install for MJ is because of an issue one of my users brought up that they can't install it using Chocolatey if they don't have admin rights.

Looking at that package, it requires admin rights to install. Seems to me that adding a specific dependency to a .NET 5 desktop runtime package that explicitly requires admin rights would block installation for people who don't have admin rights but do have the .NET 5 desktop runtime already, which is problematic. Thoughts?

Reply:
It will be updated to 5 next weekend, since it is needed by multiple manifests.

So rather shipped invalid application, which needs specific dependency to work and not notify user by any mean, so they will struggle with it and opening issues about it, instead of suggesting it??

Reply:
Well, if you run it on a machine without the .NET 5 desktop runtime installed, you get a dialog telling you to install it complete with a link to the download site. Seems friendlier than telling people for whom it would otherwise work (since they already have the runtime) that they need to install one again via Scoop first so that it'll recognize the dependency. But as you like.

Next question: I'm looking at these dependencies, and I'm not seeing a way to indicate which version of a package you depend on?

Reply:
...okay, no, wait. You can only depend on the latest manifest version, because the latest manifest version is the only one in the bucket.

Which means that you _will_ need a separate manifest to install the .NET 5 runtime, because .NET Core-and-now-.NET runtimes aren't backwards compatible across major version numbers, and so updating the windowsdesktop-runtime package to point at a non-.NET Core 3 runtime would break everything currently depending on it.